test/built-ins/Boolean/toboolean.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2017 Josh Wolfe. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+esid: sec-boolean-constructor-boolean-value
+description: ToBoolean type coercion tests via Boolean()
+info: >
+  Boolean ( value )
+
+  1. Let b be ToBoolean(value).
+  2. If NewTarget is undefined, return b.
+
+features: [BigInt, Symbol, Symbol.toPrimitive]
+includes: [typeCoercion.js]
+---*/
+
+testCoercibleToBooleanFalse(function(falsy) {
+  assert.sameValue(Boolean(falsy), false);
+});
+
+testCoercibleToBooleanTrue(function(truthy) {
+  assert.sameValue(Boolean(truthy), true);
+});
```

Comment:
This should also be placed in 2 different files, at least. We are not trying to condense everything in the same test file. This is also not helpful if anything is failing.